tardis/widgets/line_info.py

```diff
@@ -0,0 +1,749 @@
+"""Class to create and display Line Info Widget."""
```

Comment:
Might be good to say what the line info widget is

Reply:
It's right next in following line.